packages/client/src/checkout/__tests__/patchCheckoutOrderDeliveryBundles.test.ts

```diff
@@ -0,0 +1,71 @@
+import * as checkoutClient from '../index.js';
+import {
+  checkoutOrderId,
+  deliveryBundleId,
+} from 'tests/__fixtures__/checkout/index.mjs';
+import client from '../../helpers/client/index.js';
+import fixtures from '../__fixtures__/patchCheckoutOrderDeliveryBundles.fixtures.js';
+import mswServer from '../../../tests/mswServer.js';
+import type { PatchCheckoutOrderDeliveryBundlesData } from '../types/index.js';
+
+describe('patchCheckoutOrderDeliveryBundles', () => {
+  beforeEach(() => jest.clearAllMocks());
+
+  const spy = jest.spyOn(client, 'patch');
+  const expectedConfig = undefined;
+  const data: PatchCheckoutOrderDeliveryBundlesData = [
+    {
+      op: 'replace',
+      path: '/0/isSelected',
+      value: 'true',
+    },
+  ];
+  const urlToBeCalled = `/checkout/v1/orders/${checkoutOrderId}/deliveryBundles/${deliveryBundleId}`;
+
+  it('should handle a client request successfully', async () => {
+    mswServer.use(fixtures.success());
+
+    await expect(
+      checkoutClient.patchCheckoutOrderDeliveryBundles(
+        checkoutOrderId,
+        deliveryBundleId,
+        data,
+      ),
+    ).resolves.toBe(204);
+    expect(spy).toHaveBeenCalledWith(urlToBeCalled, data, expectedConfig);
+  });
+
+  // it('should handle a client request successfully with data', async () => {
```

Comment:
Check the comments here as well.

Reply:
same as here https://github.com/Farfetch/blackout/pull/860#discussion_r1323142915